core/controllers/reader.py

```diff
@@ -146,7 +176,17 @@ class ExplorationPageEmbed(base.BaseHandler):
 
     @require_playable
     def get(self, exploration_id):
-        """Handles GET requests."""
+        """Handles GET requests.
+        The require_playable decorator ensures the learner can play the
+        exploration with the exploration_id.
+
+        Args:
+            exploration_id: str. The id of the exploration requested.
+
+        Raises:
+            PageNotFound: The exploration item does not exist or is not
```

Comment:
exploration item --> exploration

What is the difference between "does not exist" and "is not available" -- i.e. why doesn't just the former suffice?